nuclear/cmake/Modules/NUClearCompilerSettings.cmake

```diff
@@ -23,7 +23,4 @@ if(NOT MSVC)
   set(CMAKE_CXX_STANDARD 17)
   set(CMAKE_CXX_STANDARD_REQUIRED ON)
   set(CMAKE_POSITION_INDEPENDENT_CODE ON)
-endif()
-
-# Set OpenCL Target version
-add_compile_definitions(CL_TARGET_OPENCL_VERSION=120)
\ No newline at end of file
+endif()
```

Comment:
No newline at end of file (although I'm sure `cmake-format` is yelling at you for this already)